Deprecate apiserver-proxy and iam-kubeconfig (#13)
Demonsthere/kyma: docs/security/05-02-iam-kubeconfig-service.md

type: Configuration
---

> >**CAUTION:** The IAM Kubeconfig component (iam-kubeconfig) is entering deprecation phase - a simpler solution is planned.

Reviewer: ```suggestion
>**CAUTION:** The IAM Kubeconfig component (iam-kubeconfig) is deprecated and will be replaced with a different mechanism.
```
